lib/connection.dart

```diff
@@ -0,0 +1,30 @@
+/*
+ * This file is part of the Embla Flutter app
+ * Copyright (c) 2020-2022 Miðeind ehf. <mideind@mideind.is>
+ * Original author: Sveinbjorn Thordarson
```

Comment:
Header copy paste. Ekki setja nafnið mitt á kóðann ykkar :). Settu bara nöfn ykkar, eða ekkert, hvort sem þú vilt.

Reply:
Og rétt ártal betra, þ.e. 2022, ekki 2020-2022.

Reply:
["If this code works, it was written by Paul DiLascia. If not, I don't know who wrote it."](https://en.wikipedia.org/wiki/Paul_DiLascia)